server/app/services/program/ProgramServiceImpl.java

```diff
@@ -345,6 +346,31 @@ public ProgramDefinition moveBlock(
         .join();
   }
 
+  @Override
+  @Transactional
+  public StatusDefinitions getStatuses(long programId) throws ProgramNotFoundException {
```

Comment:
ProgramDefinition already contains the status information. Given that, wouldn't a caller likely use one of the overloads of `getProgramDefinition`?

Reply:
Ah thanks, removed.